osu.Game.Rulesets.Catch/Edit/CatchHitObjectComposer.cs

```diff
@@ -132,7 +132,7 @@ public override SnapResult FindSnappedPositionAndTime(Vector2 screenSpacePositio
 
             result.ScreenSpacePosition.X = screenSpacePosition.X;
 
-            if (snapType.HasFlagFast(SnapType.Grids))
+            if (snapType.HasFlagFast(SnapType.GlobalGrids))
```

Comment:
This doesn't look right. The outer check has `GlobalGrids` but the inner logic concerns the distance snap grid. Isn't that a `RelativeGrid`? What am I missing here?

Reply:
More like what did *I* miss.